flowmachine/flowmachine/core/server/query_schemas/aggregate_network_objects.py

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marshmallow import Schema, fields, post_load
+from marshmallow.validate import OneOf, Length
+
+from flowmachine.features import AggregateNetworkObjects
+from .base_exposed_query import BaseExposedQuery
+from .custom_fields import AggregationUnit
+
+__all__ = ["AggregateNetworkObjectsSchema", "AggregateNetworkObjectsExposed"]
+
+
+class AggregateNetworkObjectsSchema(Schema):
+
+    start_date = fields.Date(required=True)
+    end_date = fields.Date(required=True)
+    aggregation_unit = AggregationUnit()
+
+    @post_load
+    def make_query_object(self, params):
+        return AggregateNetworkObjectsExposed(**params)
+
+
+class AggregateNetworkObjectsExposed(BaseExposedQuery):
+    def __init__(self, *, start_date, end_date, aggregation_unit):
+        # Note: all input parameters need to be defined as attributes on `self`
+        # so that marshmallow can serialise the object correctly.
+        self.start_date = start_date
+        self.end_date = end_date
+        self.aggregation_unit = aggregation_unit
+
+    @property
+    def _flowmachine_query_obj(self):
+        """
+        Return the underlying flowmachine aggregate_network_objects object.
+
+        Returns
+        -------
+        Query
+        """
+        return AggregateNetworkObjects(
```

Comment:
Basic idea here is fine, but need to create the `TotalNetworkObjects` object and return the result of calling `aggregate` on it. Or, wait for a merge on #609 which includes a refactor of `TotalNetworkObjects`, because it was necessary to making `spatial_aggregate` work!